flaskapp.py

```diff
@@ -105,6 +105,7 @@ class NetworkView(object):
     rules = []
     use_logic_filter_rules = False
     logic_filter_rules = []
+    flags = 'STRING'
```

Comment:
Is this attribute used anywhere?

Reply:
Yes this field is supported for eadefinition object with pr https://github.com/infobloxopen/infoblox-ansible/pull/235, This field contains extensible attribute flags.

Reply:
Waiting for this PR to merge, So CI will pass for integration test added in https://github.com/infobloxopen/infoblox-ansible/pull/246